run panics if lldb is not installed and no output is produced..

Index: rust/src/bootstrap/src/core/build_steps/test.rs
===================================================================
--- rust.orig/src/bootstrap/src/core/build_steps/test.rs
+++ rust/src/bootstrap/src/core/build_steps/test.rs
@@ -1779,7 +1779,11 @@ NOTE: if you're sure you want to do this
             .ok();
         if let Some(ref vers) = lldb_version {
             cmd.arg("--lldb-version").arg(vers);
-            let lldb_python_dir = run(Command::new(lldb_exe).arg("-P")).ok();
+            let lldb_python_dir = Command::new(lldb_exe)
+                .arg("-P")
+                .output()
+                .map(|output| String::from_utf8_lossy(&output.stdout).to_string())
+                .ok();
             if let Some(ref dir) = lldb_python_dir {
                 cmd.arg("--lldb-python-dir").arg(dir);
             }
